tektoncd/pipelines-as-code
# Support Public Cloud Bitbucket

Add support for bitbucket :

- pull_request event
- acl support ownership from workspace.
- acl support ownership from prow OWNERS file.
- acl support with /ok-to-test
- push support.
- /retest support.
- update status if fail/success
- add comment to PR on fail/success with task breakdown.
- by default unless disabled we can check if the webhook comes from
  https://ip-ranges.atlassian.com/

Signed-off-by: Chmouel Boudjnah <chmouel@redhat.com>
